Subscribers/Article.php

```diff
@@ -1,11 +1,14 @@
 <?php
 
 namespace ShopwarePlugins\Connect\Subscribers;
+use ShopwarePlugins\Connect\Components\ErrorHandler;
 use Shopware\Connect\Struct\Change\FromShop\MakeMainVariant;
 use Shopware\Models\Customer\Group;
 use Shopware\Connect\Gateway;
 use Shopware\Components\Model\ModelManager;
 use ShopwarePlugins\Connect\Components\ConnectExport;
+use Shopware\Models\Article\Article as Product;
```

Comment:
Yeah, u r rigth, but the Subscriber class is also named Article so there is a conflict, I will rename it to ArticleModel